src/components/bar.cpp

```diff
@@ -415,19 +415,12 @@ void bar::parse(string&& data, bool force) {
 
   m_renderer->end();
 
-  const auto check_dblclicks = [&]() -> bool {
-    if (m_action_ctxt->has_double_click()) {
-      return true;
-    }
-
-    for (auto&& action : m_opts.actions) {
-      if (static_cast<int>(action.button) >= static_cast<int>(mousebtn::DOUBLE_LEFT)) {
-        return true;
-      }
+  m_dblclicks.clear();
+  for (auto&& action : m_opts.actions) {
```

Comment:
As far as I can tell, `m_opts.actions` is only set once in `bar::bar` so we could put this loop there. Tough I'm not sure if `m_opts.actions` might change somwhere else

Reply:
Doesn't really matter. Leave it here for now.